package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eoapi-cdk",
-  "version": "9.1.1",
+  "version": "9.2.0-rc.8",
```

Comment:
Seems strange to jump to rc.8? Don't know how packaging/releasing works for this repo, though.

Reply:
Yeah, I've been setting rc and then issuing pre-releases of this branch to test this with a project.  Will revert back to `9.1.1`, CICD should set the version properly.

Reply:
```suggestion
  "version": "9.1.1",
```